.circleci/config.yml

```diff
@@ -354,7 +354,7 @@ jobs:
     executor: linux
     steps:
       - setup:
-          scarb_version: 0.5.0-alpha.0
+          scarb_version: 0.5.0-alpha.4
```

Comment:
There is `0.5.0` already

Reply:
Yeah, but we'd have to bump our cairo fork as well. I think we'll be bumping it soon anyway, once our other prs get merged.